public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -180,6 +175,7 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
   ) {
     try {
       setIsLoadingAnomalyResults(true);
+      const resultIndex = get(props, 'detector.resultIndex', '');
```

Comment:
same thing here - maybe move to the top of the file so it only needs to be declared once

Reply:
Done